pyant/phased_finite_cylindrical_parabola.py

```diff
@@ -119,7 +119,7 @@ def gain(self, k, polarization=None, ind=None):
         x = height/wavelength*np.sin(theta)  # sinc component (longitudinal)
         y = width/wavelength*np.sin(phi - phase_steering)   # sinc component (transverse)
         G = np.sinc(x)*np.sinc(y) # sinc fn. (= field), NB: np.sinc includes pi !!
-        G = G*np.cos(phi)*np.cos(phase_steering) # density (from spherical integration)
```

Comment:
did we find out why the cos() terms should not be there? or why they were there in the first place?

Reply:
sorting out later